src/Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
```

Comment:
is it possible to have a version without having to install external modules, only ruby?
We can live without the commit name check for now and the colors

Reply:
but i thought commit name check was required. Will it be required later on ?

Reply:
as we are trying to push contributors to use phabricator instead of github, names should be fine later.
It is not one of the most important change and we can do without it.

Reply:
file to be removed?